zenodo/modules/deposit/jsonschemas/deposits/records/record-v1.0.0.json

```diff
@@ -233,12 +233,12 @@
               "type": "integer"
             },
             "type": {
-              "description": "Type of the PID, in correspondece with PID's 'pid_type' property.",
+              "description": "Type of the PID, in correspondence with PID's 'pid_type' property.",
```

Comment:
```suggestion
              "description": "Type of the PID, in correspondece with PID's 'pid_type' property.",
```

Reply:
uncommented the two relevant sed lines in my ticket's shell script